cmake/Modules/LocalRav1e.cmake

```diff
@@ -51,6 +51,9 @@ else()
             GIT_REPOSITORY https://github.com/lu-zero/cargo-c.git
             GIT_TAG "${AVIF_CARGOC_GIT_TAG}"
             GIT_SHALLOW ON
+            # Patch from https://github.com/lu-zero/cargo-c/pull/453
+            PATCH_COMMAND sed -i.bak -e "s/0.10.11+cargo-0.86.0/0.10.12+cargo-0.87.0/g" Cargo.toml && sed -i.bak -e
```

Comment:
can you do a git patch file instead? sed can be annoying cross paltform as we've seen in the past

Reply:
Done